### PR TITLE
Toggling 3 detections and announcing 2 new detections

### PR DESCRIPTION
**Pull request by unknownhad**
Toggling 3 detections and announcing 2 new detections

### CODE REVIEW

File: content/waf/change-log/scheduled-changes.md [hunk 1]
```diff
@@ -24,42 +24,22 @@ layout: list
...
   <tbody>
     <tr>
-      <td>2022-07-11</td>
       <td>2022-07-18</td>
+      <td>2022-07-25</td>
       <td>Cloudflare Specials</td>
-      <td>...2753531e</td>
-      <td>100533</td>
-      <td>NoSQL - Injection</td>
+      <td>...N/A</td>
```
Comment (unknownhad) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
      <td>...1651d0c8</td>
```

File: content/waf/change-log/scheduled-changes.md [hunk 2]
```diff
@@ -24,42 +24,22 @@ layout: list
...
-      <td>Block</td>
-    </tr>
-    <tr>
-      <td>2022-07-11</td>
-      <td>2022-07-18</td>
-      <td>Cloudflare Specials</td>
-      <td>...aa290ad9</td>
-      <td>100135D</td>
-      <td>XSS - JS On Events</td>
+      <td>...N/A</td>
```
Comment (unknownhad) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
      <td>...6a648210</td>
```

**Review by gpgabriel — APPROVED** [review]

**Review by pedrosousa — APPROVED** [review]

**Review by vs-mg — COMMENTED** [review]
Added 100533

File: content/waf/change-log/2022-07-18.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      <td>Block</td>
+    </tr>
+    <tr>
+      <td>Cloudflare Specials</td>
+      <td>...aa290ad9</td>
+      <td>100135D</td>
+      <td>XSS - JS On Events</td>
+      <td>N/A</td>
+      <td>Block</td>
+    </tr>
```
Comment (unknownhad) [inline content/waf/change-log/2022-07-18.md:55]:
```suggestion
    </tr>
    <tr>
      <td>Cloudflare Specials</td>
      <td>... 2753531e</td>
      <td>100533</td>
      <td>NoSQL - Injection</td>
      <td>N/A</td>
      <td>Block</td>
    </tr>
   
```

**Pull request merged**